Crash in AmazonWebkitFactories

This is a crash on the LiveTest 1.2 version - app crashes on startup.

Building and installing from source works for most people.

```
02-15 16:23:02.863 21774-21774/? W/com.amazon.android.webkit.AmazonWebKitFactories: Create WebKitFactoryByName failed: com.amazon.android.webkit.embedded.EmbeddedWebKitFactory
02-15 16:23:02.863 21774-21774/? W/com.amazon.android.webkit.AmazonWebKitFactories: Create WebKitFactoryByName failed: com.amazon.android.webkit.android.AndroidWebKitFactory
02-15 16:23:02.864 21774-21774/? D/AndroidRuntime: Shutting down VM
02-15 14:20:29.767 7097-7097/? E/AndroidRuntime: FATAL EXCEPTION: main
Process: org.mozilla.tv.firefox, PID: 7097
java.lang.ExceptionInInitializerError
    at com.amazon.android.webkit.AmazonWebKitFactories.getDefaultFactory(AmazonWebKitFactories.java:132)
    at org.mozilla.focus.activity.MainActivity.initAmazonFactory(MainActivity.java:221)
    at org.mozilla.focus.activity.MainActivity.onCreateMainActivity(MainActivity.java:72)
    at org.mozilla.focus.activity.MainActivity.onCreate(MainActivity.java)
    at android.app.Activity.performCreate(Activity.java:6010)
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1122)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2320)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2435)
    at android.app.ActivityThread.access$800(ActivityThread.java:160)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:135)
    at android.app.ActivityThread.main(ActivityThread.java:5483)
    at java.lang.reflect.Method.invoke(Native Method)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:984)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
 Caused by: java.lang.RuntimeException: Unable to initialize webkit factory
    at com.amazon.android.webkit.AmazonWebKitFactories$Implementation.<clinit>(AmazonWebKitFactories.java:72)
    at com.amazon.android.webkit.AmazonWebKitFactories.getDefaultFactory(AmazonWebKitFactories.java:132) 
    at org.mozilla.focus.activity.MainActivity.initAmazonFactory(MainActivity.java:221) 
    at org.mozilla.focus.activity.MainActivity.onCreateMainActivity(MainActivity.java:72) 
    at org.mozilla.focus.activity.MainActivity.onCreate(MainActivity.java) 
    at android.app.Activity.performCreate(Activity.java:6010) 
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1122) 
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2320) 
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2435) 
    at android.app.ActivityThread.access$800(ActivityThread.java:160) 
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321) 
    at android.os.Handler.dispatchMessage(Handler.java:102) 
    at android.os.Looper.loop(Looper.java:135) 
    at android.app.ActivityThread.main(ActivityThread.java:5483) 
    at java.lang.reflect.Method.invoke(Native Method) 
    at java.lang.reflect.Method.invoke(Method.java:372) 
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:984) 
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779) 
```

--- a/app/proguard-rules.pro
+++ b/app/proguard-rules.pro
@@ -32,6 +32,9 @@
 #   public *;
 #}
 
+-keep public class com.amazon.android.webkit.android.AndroidWebKitFactory { public *; }
+-keep public class com.amazon.android.webkit.embedded.EmbeddedWebKitFactory { public *; }
+
 # Uncomment this to preserve the line number information for
 # debugging stack traces.
 #-keepattributes SourceFile,LineNumberTable

